### DISCUSSION TITLE
Updating the rules of engagement for 2023 GOA BTS

### DISCUSSION THREAD

**Discussion by zoyafuso-NOAA** [afsc-gap-products/StationAllocationAIGOA#5] [General]
@Lewis-Barnett-NOAA @Ned-Laman-NOAA @MargaretSiple-NOAA 
**Main question**: do we need to modify the current shelf/edge/slope depth mismatch rules with the new stratification? 

These rules were made because of the mismatch between the bathymetry used to define the historical (i.e., pre-2023) stratification and the actual bathymetry measured when trawling. These rules are: 

Rule 1: Gully and Slope strata shall be trawled at their prescribed depths, even if it means moving the station location. Why? Because these typically narrow strata have depth boundaries that are drawn incorrectly (e.g., do not follow the actual depth contours) and we might completely miss trawling at these depths

Rule 2: Shelf strata shall be towed at their prescribed station locations, even it if means towing shallower or deeper than prescribed. Why? Because far inside of their boundaries these typically wide strata have numerous shallower or deeper areas which are orphaned from their (continuous) depth zones

Rule 3: The exception to Rule 2 happens when a shelf station is near (1 or 2 miles) ANY other stratum depth boundary and it is clear that the stratum depth line is drawn incorrectly – then it must be trawled at the prescribed depth

**Comment by zoyafuso-NOAA** [comment]
This series of excerpts is where we left off on this discussion point from an email thread (Subject: GOA survey design optimization update: feedback solicitation) from earlier this year:

**Ned (January 24th 2022):**
Since we will not have perfectly accurate bathymetry anytime soon, we need a set of rules to resolve conflicts between assigned station location and stratum depth and the actual depth under the vessel when you get to the station.
…
We will either need to stick to these and try to re-stratify in a way that maintains the validity of these rules OR we will need to anticipate changes we need and re-write the Rules of Engagement so that we can prosecute the survey under a new stratification design.

**Lewis (January 26th 2022)**
Interesting point regarding station location vs depth as the criteria for determining which stratum an observation should be in.  I can see the merits of either approach, but given that the boundaries here are being driven largely by the CPUE at the location of prior stations, I would think that preserving the integrity of boundaries based on location rather than depth would be more in line with the information provided by the analysis.  However, I see how this could be a problem when dealing with very narrow strata and strong depth gradients, so I'll be keen to hear thoughts on this from the rest of the team.

**Ned (January 26th 2022)**
Thanks for your response, Lewis.
I want to make sure I understand your assessment of and likely response to the location vs depth criterion for determining where a sample is trawled.

Here's scenario to help me understand what you are suggesting:

You are assigned station 1-1 in stratum 4 where the definition of the stratum depth range is 300-500m out on the slope at the shelf edge. 

You arrive at station 1-1 and find that you are at the right location based on the X on the and the stratum boundaries on the chart, but the depth is shallower at your station (250m deep) than the stratum definition.

Would you recommend sampling station 1-1 in 250m and leaving it assigned to stratum 4?

The rules of engagement indicate that we should find an alternate to station 1-1, but within stratum 4 so that we execute a trawl in the assigned 300-500m depth.

**Lewis (January 28th 2022)**
Thanks for providing an example of the alternatives.  I am obviously open to whatever the group decides, and understand the importance of these fine-scale differences in depth, but my initial opinion is that I would lean slightly toward the first scenario (sampling station 1-1 in 250m and leaving it assigned to stratum 4) to avoid too much time spent searching for alternate stations, but I would like to get a sense of how often we expect this to happen.  I'd also be happy to discuss the rationale for this further in relation to the analytical approach, but it is probably easier to explain verbally.

**Comment by Ned-Laman-NOAA** [comment]
Under the 2023 sampling design and restratification, the rules of engagement for stations will be greatly simplified.
Rule 1 (and only rule) - tow the station not the depth.
Under the new design, any station within a stratum boundary can be considered representative of that stratum whether the depth is within the depth stratum's defined range or not. Am I right on this?

**Comment by Lewis-Barnett-NOAA** [comment]
correct on rule 1 Ned

On Thu, Oct 13, 2022 at 7:47 AM Ned Laman ***@***.***> wrote:

> Under the 2023 sampling design and restratification, the rules of
> engagement for stations will be greatly simplified.
> Rule 1 (and only rule) - tow the station not the depth.
> Under the new design, any station within a stratum boundary can be
> considered representative of that stratum whether the depth is within the
> depth stratum's defined range or not. Am I right on this?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/afsc-gap-products/StationAllocationAIGOA/discussions/5#discussioncomment-3870947>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAKMJP6JGZWBAIUNSBWDAALWDAOGDANCNFSM6AAAAAAQ53KIUI>
> .
> You are receiving this because you were mentioned.Message ID:
> <afsc-gap-products/StationAllocationAIGOA/repo-discussions/5/comments/3870947
> @github.com>
>


-- 
Lewis Barnett, PhD (he/him/his)
Research Fish Biologist

NOAA Fisheries, Alaska Fisheries Science Center
7600 Sand Point Way NE, Bldg 4
Seattle, Washington 98115
Google Voice: (206) 526-4111
